en/css/README.md

```diff
@@ -70,19 +70,22 @@ Time to write some CSS! Open up the `blog/static/css/blog.css` file in your code
 
 We won't be going too deep into customizing and learning about CSS here. There is a recommendation for a free CSS course at the end of this page if you would like to learn more. 
 
-But let's do at least a little. Maybe we could change the color of our header?
+But let's do at least a little. Maybe we could change the color of our headers?
 To understand colors, computers use special codes. These codes start with `#` followed by 6 letters (A–F) and numbers (0–9). For example, the code for blue is `#0000FF`. You can find the color codes for many colors here: http://www.colorpicker.com/. You may also use [predefined colors](http://www.w3schools.com/colors/colors_names.asp), such as `red` and `green`.
 
 In your `blog/static/css/blog.css` file you should add the following code:
 
 {% filename %}blog/static/css/blog.css{% endfilename %}
 ```css
 h1 a {
-    color: #FCA205;
+    color: #C25100;
+}
+h2 a {
```

Comment:
Should this be `h1 a, h2 a` perhaps? Not sure if the current explicitness makes it easier for beginners though.

Reply:
Yeah, I wondered about that. I wasn't sure if we wanted to get into explaining having two selectors for the same code or not?

Reply:
Actually I see we have multiple selectors on the same line farther down in the file, so I made the change here.